src/shared/locales/all.js

```diff
@@ -34,6 +35,7 @@ import appLocaleVi from '@shared/locales/vi'
 import appLocaleZhCN from '@shared/locales/zh-CN'
 import appLocaleZhTW from '@shared/locales/zh-TW'
 import appLocaleUk from '@shared/locales/uk'
+import appLocaleHu from '@shared/locales/hu'
```

Comment:
Please let import also be in alphabetical order.

Reply:
fixed